packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/actions.sh

```diff
@@ -5,5 +5,55 @@
 # CWD is agoric-sdk
 here=./upgrade-test-scripts/agoric-upgrade-11
 
-# run zoe thru "null upgrade"
-$here/zoe-upgrade/zoe-upgrade-driver.sh
+# Pre-steps:
+#  * fill Wallets
+#  * install bundles
+#  * create instance of prober contract and run expecting no atomicRearrange
+#
+# Action:
+#  * upgrade Zoe and ZCF
+#
+# Finish
+#  * create instance of prober contract and run expecting to see atomicRearrange
+
+echo XXXX fill wallet XXXXXX
```

Comment:
XXX elsewhere means tech debt. please user a different character.  Or just upcase.
```suggestion
echo FILL WALLET
```

Reply:
done